frontend/app/routes/_public+/apply+/$id+/terms-and-conditions.tsx

```diff
@@ -0,0 +1,173 @@
+          >
+            {t('terms-and-conditions:terms-and-conditions.index.privacy-notice-statement-how-we-protect-your-privacy-text-part-three-part-1-link-text')}
+          </InlineLink>
+          {t('terms-and-conditions:terms-and-conditions.index.privacy-notice-statement-how-we-protect-your-privacy-text-part-three-part-2')}
+        </p>
+        <br />
+        <p>
+          {t('terms-and-conditions:terms-and-conditions.index.privacy-notice-statement-how-we-protect-your-privacy-text-part-four')}
+          <InlineLink to={`https://www.microsoft.com/en-ca/trust-center/privacy`} className="font-lato font-semibold">
+            {t('terms-and-conditions:terms-and-conditions.index.privacy-notice-statement-how-we-protect-your-privacy-text-part-four-link-text')}
+          </InlineLink>
+        </p>
+        <br />
+        <p>
+          {t('terms-and-conditions:terms-and-conditions.index.privacy-notice-statement-how-we-protect-your-privacy-text-part-five')}
+          <InlineLink to={`https://www.figma.com/file/pXiZz8YkzcMY5S5aq0B4ce/CDCP-Online-Application-Prototype?type=design&node-id=3091-43199&mode=design&t=HrfyDoQ3uFrjD1Zh-4`} className="font-lato font-semibold">
+            {t('terms-and-conditions:terms-and-conditions.index.privacy-notice-statement-how-we-protect-your-privacy-text-part-five-link-text')}
+          </InlineLink>
+        </p>
+        <br />
+        <p>
+          <strong>{t('terms-and-conditions:terms-and-conditions.index.privacy-notice-statement-who-we-can-share-your-information-with-title')}</strong>
+        </p>
+        <p>{t('terms-and-conditions:terms-and-conditions.index.privacy-notice-statement-who-we-can-share-your-information-with-text-part-one')}</p>
+        <br />
+        <p>{t('terms-and-conditions:terms-and-conditions.index.privacy-notice-statement-who-we-can-share-your-information-with-text-part-two')}</p>
+        <br />
+        <p>{t('terms-and-conditions:terms-and-conditions.index.privacy-notice-statement-who-we-can-share-your-information-with-text-part-three')}</p>
+        <br />
+        <strong>
+          <p>{t('terms-and-conditions:terms-and-conditions.index.privacy-notice-statement-what-happens-if-you-dont-give-us-your-information-title')}</p>
+        </strong>
+        <p>{t('terms-and-conditions:terms-and-conditions.index.privacy-notice-statement-what-happens-if-you-dont-give-us-your-information-text')}</p>
+      </CollapsibleDetails>
+      <br /> <p>{t('terms-and-conditions:terms-and-conditions.index.apply-now.text')}</p>
+      <br />
+      <div className="flex flex-wrap items-center gap-3">
+        <ButtonLink id="back-button" to={`/apply`}>
+          {t('terms-and-conditions:terms-and-conditions.index.apply-now.back-button')}
+        </ButtonLink>
+        <ButtonLink id="continue-button" to={`/apply/${id}/personal-info`}>
+          {t('terms-and-conditions:terms-and-conditions.index.apply-now.start-button')}
+        </ButtonLink>
```

Comment:
```suggestion
        <ButtonLink id="back-button" to={`/apply`}>
          <FontAwesomeIcon icon={faChevronLeft} className="me-3" />
          {t('terms-and-conditions:terms-and-conditions.index.apply-now.back-button')}
        </ButtonLink>
        <ButtonLink id="continue-button" to={`/apply/${id}/personal-info`} variant="primary">
          {t('terms-and-conditions:terms-and-conditions.index.apply-now.start-button')}
          <FontAwesomeIcon icon={faChevronRight} className="ms-3" />
        </ButtonLink>
```

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/20ed263d-a8dc-4930-ae39-35c7e820da80)

Reply:
Fixed: ed62887534ecb1a1d9810f348c88031cfc813d39